src/ScottPlot4/ScottPlot/Plottable/SignalPlotBase.cs

```diff
@@ -21,6 +21,8 @@ public abstract class SignalPlotBase<T> : IPlottable, IHasLine, IHasMarker, IHig
         public int YAxisIndex { get; set; } = 0;
         public bool IsVisible { get; set; } = true;
         public bool StepDisplay { get; set; } = false;
+        public bool Smooth { get; set; } = false;
+        public double SmoothTension { get; set; } = 0.5f;
```

Comment:
This is always going to be converted to a float before use, so we can probably just expose a float rather than exposing a double and casting it each time.

Reply:
I'm okay exposing the double and later casting to float. 

It's frustrating to always require users to type `.5f` instead of just `.5`